### DISCUSSION TITLE
Does HiGHS simplex provide "quick start" steepest edge pricing?

### DISCUSSION THREAD

**Discussion by ambros-gleixner** [ERGO-Code/HiGHS#1177] [Q&A] [answered]
I am wondering whether HiGHS has something similar to what is called "quick start" steepest edge in many simplex codes, i.e., where norms are initialized with 1 and just updated. In cplex this is, e.g., the CPX_DPRIIND_STEEPQSTART parameter.

**Comment by ambros-gleixner** [comment]
@jajhall Could you give some brief insight into this?

**Comment by jajhall** [comment, marked as answer]
No, we don't do quick-start like this. The MIP solver saves the steepest edge weights at nodes, for use when not starting from a child of the last-solved node

**Comment by ambros-gleixner** [comment]
Thanks! (We do the same with SoPlex in SCIP.)
